app/grandchallenge/evaluation/forms.py

```diff
@@ -124,8 +126,28 @@ class Meta:
             "extra_results_columns": JSONEditorWidget(
                 schema=EXTRA_RESULT_COLUMNS_SCHEMA
             ),
+            "submissions_open": forms.DateInput(
```

Comment:
Does it make sense to use a DateInput here? I think we use a DateTimeField for both of the deadlines, so that the challenge admin can set a time of say 5pm on this date.